lib/getComponents.js

```diff
@@ -0,0 +1,46 @@
+const baseUrl = process.env.BASE_URL;
+
+async function getComponents(page, sectionUrl, type = "html") {
+  // Go to section url. Example: https://tailwindui.com/components/marketing/sections/heroes
+  await page.goto(sectionUrl);
+
+  const toggleCodeSelector = '[role=tablist] > [id^=headlessui-tabs-tab-]:nth-child(2)'
+  try {
+    // Element to toggle code snippet (hint: ../docs/hints/toggle-code.png). 
+    // The element won't exist if the user doesn't have access to the paid component.
+    await page.waitForSelector(toggleCodeSelector, { timeout: 10 * 1000 });
+  } catch (error) {
+    console.warn(`You do not have access to this section components: ${sectionUrl}`);
+  }
+
+  const components = await page.evaluate(([toggleCodeSelector, type]) => {
+    let components = [];
+    document.querySelectorAll('section[id^="component-"]').forEach((el) => {
+      const toggleCode = el.querySelector(toggleCodeSelector);
+      if (!toggleCode) {
+        return
+      }
```

Comment:
Allow the script to keep going when the code block is locked (user has no access to it). 

[Example component](https://tailwindui.com/components/marketing/sections/heroes#component-e5c15b9d1a503738005e35302043c802) ("Get the code" indicates the code block is locked):

![image](https://user-images.githubusercontent.com/7823011/199479860-485c00e9-5c3e-4ccb-a4c4-ab35d371fe30.png)